package.json

```diff
@@ -93,5 +93,9 @@
   },
   "files": [
     "dist"
-  ]
+  ],
+  "volta": {
+    "node": "12.16.0",
```

Comment:
usually we pin to node versions that are the lowest we technically support, this helps us catch missing features during development

Reply:
Ok I downgraded this to node 8.